src/EventListener/CacheControlListener.php

```diff
@@ -16,8 +16,16 @@
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\HttpKernel\Event\FilterResponseEvent;
+use Symfony\Component\HttpKernel\Event\ResponseEvent;
+use Symfony\Component\HttpKernel\Kernel;
 use Symfony\Component\HttpKernel\KernelEvents;
 
+if (Kernel::VERSION >= 5) {
```

Comment:
beware, version is a string, what's about using `kernel::MAJOR_VERSION >= 5` or `kernel::VERSION_ID >= 50000`?

Reply:
ups, yes i wanted to use MAJOR_VERSION here, thanks!